WeakAuras/WeakAuras.toc

```diff
@@ -33,6 +33,10 @@ embeds.xml
 # #@end-version-classic@
 #@end-non-version-retail@
 
+#@version-retail@
+# Libs\LibSpecialization\LibSpecialization.lua
+#@end-version-retail@
```

Comment:
This break using non-packaged version on retail, and if un-commented it break non-packaged version on other versions. That would be annoying for dev (currently work on bcc/wrath).

I think the multitoc PR has to be resurrected first.